### PR TITLE
Restructured dashboard for review by TWG

### PR DESCRIPTION
**Pull request by DBees**

### CODE REVIEW

File: dashboard/Dashboard_SpreadSheetFriendly.md [hunk 1]
```diff
@@ -1,69 +1,94 @@
...
-|	CORE-V WALLY 	|	TRL 4	|	PC	|	[David Harris](https://github.com/davidharrishmc)	|	[cvw](https://github.com/openhwgroup/cvw)	|	23.02.27 (A)	|		|		|		|		|	[Project Concept](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-WALLY/Project-Concept-OpenHW-Wally.pdf)	|		|		|		|																										
-|	CORE-V CVA6-H	|	TRL 5	|	PC	|	"Bruno Sa, Sandro Pinto"	|	integrated within CVA6	|	23.02.27 (A)	|		|		|		|		|	[CVA6-H PC presentation](https://docs.google.com/presentation/d/1mkl5KSRtpkGT18YbUadwtFZjMNdz3euX/edit#slide=id.g1b965471e2e_0_515)	|		|		|		|																										
+|	Name				|	TRL Target	|	Status	|	TPL or PM																		|	Repo													|	PC			|	PL			|	PA		| PF Estimate 	|	Project Docs	|	RTD	|	Project Plan	|	Project Board	|
```
Comment (MikeOpenHWGroup) [inline dashboard/Dashboard_SpreadSheetFriendly.md:23]: This table does not render properly on GitHub.

File: dashboard/Dashboard_SpreadSheetFriendly.md [hunk 2]
```diff
@@ -1,69 +1,94 @@
...
-|	CORE-V-MCU-DEVKIT	|	TRL 4	|	PL	|	[Joseph Julicher](https://github.com/n9wxu)	|	[core-v-mcu devkit](https://github.com/openhwgroup/core-v-mcu-devkit)	|	21.12.12 (A)	|	22.06.27 (A)	|		|		|		|	[Project Concept](https://github.com/openhwgroup/programs/blob/master/Project-Descriptions-and-Plans/CORE-V-MCU-DEV-KIT/CORE-V_DEV-KIT_project_concept_approved.md) <br /> + [Project Launch](https://docs.google.com/document/d/15v-1zsUEahpGljZNWOrmNEDTp1RlQCiLntFuJ-Hcw0I/edit)	|		|		|		|																										
-|	CORE-V-TRUSTED-MCU	|	TRL 7 (CPU)	|	PC	|	Abdoulaye Berthe	|		|	22.07.25 (A)	|		|		|		|		|	[CORE-V-MCU2 Project Concept](https://docs.google.com/document/d/1nW9rFAFHhNkNi53khY3hhynXaHIm4OqJ/edit)	|		|		|		|																										
+## Active HW TG Projects	
+																																																					
+|	Name				| Status	|	TPL or PM																																|	Repo																	|	PC		|	PL		|	PA		|	PF Estimate	|	Project Docs																																																																						|	RTD																				|	Project Plan			|	Project Board	|
```
Comment (MikeOpenHWGroup) [inline dashboard/Dashboard_SpreadSheetFriendly.md:38]: This table does not render properly on GitHub.

**Review by MikeOpenHWGroup — CHANGES_REQUESTED** [review]
HI @DBees, the two Tables indicated do not render properly on GitHub.  You can see the results [here](https://github.com/DBees/core-v-docs/blob/updateDashboard/dashboard/Dashboard_SpreadSheetFriendly.md#active-cores-tg-projects).

File: dashboard/Dashboard_SpreadSheetFriendly.md (hunk 2 above)
Comment (DBees) [reply dashboard/Dashboard_SpreadSheetFriendly.md:38]: Huh @MikeOpenHWGroup These render properly on a chrome markdown viewer. Not sure the issue...

**Review by MikeOpenHWGroup — APPROVED** [review]
Removing the gap fixed the table.

**Pull request merged**